integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringmanagement/FindPowerFailureEventsReads.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2016 Smart Society Services B.V.
+ * Copyright 2020 Smart Society Services B.V.
```

Comment:
Copyright statement is incorrect. Earlier review comment was not completely dealt with.